src/common/Auth.js

```diff
@@ -42,10 +83,9 @@ function useGoogleAuthProvider() {
         const profile = credentials.getBasicProfile();
         const id = profile.getId(); // Do not send to your backend! Use an ID token instead.
         const name = profile.getName();
-        const image = profile.getImageUrl();
         const email = profile.getEmail(); // This is null if the 'email' scope is not present.
-        setAuthCookie({id, name, image, email});  // theoretically, this should setUser as well, since we added a listener to it
-        setUser({id, name, image, email});
+        setAuthCookie({id, name, email});  // theoretically, this should setUser as well, since we added a listener to it
```

Comment:
This is a breaking change in Profile.js. Please fix by removing line 9 in Profile.js